kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/RawCustomResourceOperationsImpl.java

```diff
@@ -34,6 +34,7 @@
 import okhttp3.Request;
 import okhttp3.RequestBody;
 import okhttp3.Response;
+import org.apache.commons.lang.StringUtils;
```

Comment:
commons-lang is an optional dependency.
We shouldn't use it unless we handle the case that its not present.

In any case we have our internal StringUtils called Strings and should always use that.

Reply:
Ok, fixing it right now